speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/mainmenu.cpp

```diff
@@ -20,63 +20,65 @@
 #include <tgfclient.h>
 
 #include <racescreens.h>
-#include <playerconfig.h>
+// SIMULATED DRIVING ASSISTANCE: REMOVED PLAYER CONFIG, ADDED CONTROLCONGFIG
+#include <controlconfig.h>
 
 #include "mainmenu.h"
 #include "exitmenu.h"
 #include "optionsmenu.h"
 #include "creditsmenu.h"
 #include "DataSelectionMenu.h"
 
-
-static void *MenuHandle = 0;
+// SIMULATED DRIVING ASSISTANCE: RENAMED MenuHandle
+static void *MainMenuHandle = 0;
```

Comment:
Was there a technical reason for this rename?

Reply:
well I first thought I needed it as a global variable, but I fixed it without that, so it not necessary anymore. I should porbably revert that